pages/congresistas/index.js

```diff
@@ -0,0 +1,9 @@
+import SidebarLayout from 'components/layout/SidebarLayout';
+
+export default function Congresistas() {
```

Comment:
```suggestion
export default function Congresspeople() {
```
Sí recomendaría mantener los nombres en inglés, igual para los directorios. Vamos a necesitar un diccionario / glosario de traducciones.